app/build.gradle

```diff
@@ -4,22 +4,20 @@ apply plugin: 'com.chaquo.python'
 
 apply plugin: 'kotlin-android'
 
-apply plugin: 'kotlin-android-extensions'
-
 android {
-    compileSdkVersion 30
-    buildToolsVersion "29.0.2"
+    compileSdk 34
     defaultConfig {
         applicationId "com.mdvmeijer.chitchat"
-        minSdkVersion 16
-        targetSdkVersion 30
+        minSdkVersion 27
+        targetSdkVersion 34
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         ndk {
-            abiFilters "armeabi-v7a", "x86"
+            abiFilters "arm64-v8a"
```

Comment:
Please add x86 to this so that it works with emulators running on x86 machines

Reply:
Sure. The side effect though is that this creates a longer building time.